src/main/java/io/jenkins/pluginhealth/scoring/probes/UpForAdoptionProbe.java

```diff
@@ -37,15 +39,20 @@
 @Component
 @Order(value = UpForAdoptionProbe.ORDER)
 public class UpForAdoptionProbe extends Probe {
+    private static final Logger LOGGER = LoggerFactory.getLogger(UpForAdoptionProbe.class);
     public static final int ORDER = DeprecatedPluginProbe.ORDER + 1;
 
     @Override
     public ProbeResult doApply(Plugin plugin, ProbeContext context) {
         final UpdateCenter updateCenter = context.getUpdateCenter();
-        if (updateCenter.plugins().get(plugin.getName()).labels().contains("adopt-this-plugin")) {
+        final io.jenkins.pluginhealth.scoring.model.updatecenter.Plugin pl = updateCenter.plugins().get(plugin.getName());
+        if (pl == null) {
+            LOGGER.info("Couldn't not find {} in update-center", plugin.getName());
+            return ProbeResult.failure(key(), "Plugin is not in the update-center");
```

Comment:
```suggestion
            return ProbeResult.failure(key(), "This plugin is not in the update-center");
```
(nit, to be similar to the other messages)